chapter2/tests/t20/terraform/base/efs.tf

```diff
@@ -0,0 +1,26 @@
+data "aws_availability_zones" "available" {}
```

Comment:
I think you should use your availability zones from vpc.tf file (`all`)

Reply:
> I think you should use your availability zones from vpc.tf file (`all`)

@kseniyashaydurova 
Done